src/components/CategoryItem.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { setIsSelected } from '../redux';
+
+export const CategoryItem = ({ category, index }) => {
+  const dispatch = useDispatch();
+
+  const selectCategory = (name) => {
+    dispatch(setIsSelected(name));
+  };
+
+  return (
+    <div className='category' onClick={() => selectCategory(category.name)}>
+      <button className='status-btn'>
+        {category.isSelected ? (
+          <i className='fa fa-smile-o'></i>
```

Comment:
For Html elements without content, you can use `<i className='fa fa-smile-o'/>` the short notation =)